### PR TITLE
Add module files

### PR DESCRIPTION
**Pull request by lazyCode-404**
### Things implemented in this milestone:

- Created a new dir
- Divided code into modules
- Displayed the date below the navigation bar using Luxon

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  bookList.classList.remove('dontShow');
+  addNew.classList.add('dontShow');
+  contact.classList.add('dontShow');
+}
+
+function showContact() {
+  contact.classList.remove('dontShow');
+  bookList.classList.add('dontShow');
+  addNew.classList.add('dontShow');
+}
```
Comment (Mre55) [inline index.js]: As of the project, requirement let's update these functions to use `arrow` syntax.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @BwalyaNonde,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @BwalyaNonde  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lazyCode-404** [comment]
Thank you so much for approving the project.

**Pull request merged**